dev-tools/src/main/resources/forbidden/core-signatures.txt

```diff
@@ -99,3 +99,7 @@ com.google.common.base.Preconditions#checkNotNull(java.lang.Object, java.lang.Ob
 com.google.common.base.Preconditions#checkNotNull(java.lang.Object, java.lang.String, java.lang.Object[])
 com.google.common.collect.ImmutableSortedSet
 com.google.common.collect.Queues
+com.google.common.util.concurrent.ListenableFuture
+com.google.common.util.concurrent.SettableFuture
+com.google.common.util.concurrent.Futures
+com.google.common.util.concurrent.MoreExecutors
```

Comment:
Can you move these to dev-tools/src/main/resources/forbidden/all-signatures.txt as in #13523?